salt/roster/cloud.py

```diff
@@ -63,7 +63,7 @@ def targets(tgt, tgt_type='glob', **kwargs):  # pylint: disable=W0613
         ))
         preferred_ip = extract_ipv4(roster_order, ip_list)
 
-        ret[minion_id] = __opts__.get('roster_defaults', {})
+        ret[minion_id] = __opts__.get('roster_defaults', {}).copy()
```

Comment:
is there any chance any of these options in roster_defaults is or could eventually be a dictionary?

If so, we should use `copy.deepcopy()` to make sure to instantiate new versions of those dictionaries as well.

Otherwise, this sounds fine.